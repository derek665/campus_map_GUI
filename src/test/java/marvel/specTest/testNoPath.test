# test the correct output when 2 nodes have no connections

LoadGraph testNo soccer.tsv

FindPath testNo Hanover96 Valencia