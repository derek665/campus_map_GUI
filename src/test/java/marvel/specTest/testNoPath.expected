# test the correct output when 2 nodes have no connections

loaded graph testNo

path from Hanover96 to Valencia:
no path found